pkg/api/handlers/compat/containers.go

```diff
@@ -29,9 +31,11 @@ import (
 func RemoveContainer(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is the compat endpoint. We shouldn't be modifying its parameters.

Reply:
This is the only endpoint we have for removing containers.  All of the Docker parameters still work, is it invalid to use additional libpod params on a compat endpoint?

@baude @jwhonce WDYT?

Reply:
You can make it conditional which is something other endpoints are doing.  `utils.IsLibpodRequest(r)` returns a boolean that indicates if the request was made against a `libpod/...` endpoint.